app/src/main/java/ch/epfl/skysync/navigation/MainGraph.kt

```diff
@@ -17,12 +17,12 @@ fun MainGraph(
     navHostController: NavHostController,
     signInLauncher: ActivityResultLauncher<Intent>,
     uid: String?,
-    timer: TimerViewModel
+    inFlightsViewModel: LocationViewModel? = null
```

Comment:
Why is `inFlightsViewModel` marked as nullable ?

Reply:
for testing purposes and to reduce refactoring of present tests